index.src.html

```diff
@@ -1916,7 +1916,9 @@
 1. If {{AudioDecoderConfig/codec}} is empty after
     [=strip leading and trailing ASCII whitespace|stripping leading and trailing ASCII whitespace=],
     return `false`.
-2. Return `true`.
+2. If the result of running  IsDetachedBuffer (described in
+    [[!ECMASCRIPT]]) on {{AudioDecoderConfig/description}} is `true`, return `false`.
```

Comment:
Given that `description` is a WebIDL value this should probably use https://webidl.spec.whatwg.org/#buffersource-detached instead.

Reply:
Thanks Peter, fixed in https://github.com/w3c/webcodecs/commit/c97f2989c9bd3e9bbca9b5a0f96a3ef029e7d7a0.